recipe/meta.yaml

```
@@ -30,13 +35,25 @@ requirements:
     - more-itertools
     - zc.lockfile
     - jaraco.collections
-    - pywin32 >=227   # [win and py<310]
-    - importlib-metadata # [py<38]
-    - simplejson
+    - pywin32 >=227  # [win and py<310]
+    - importlib-metadata  # [py<=37]
+    - pyOpenSSL
```

```suggestion
    - pyopenssl
```
the name of our package is pyopenssl